### PR TITLE
[BPK-990] Implement theming support for links

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: packages/bpk-mixins/src/mixins/_typography.scss [hunk 1]
```diff
@@ -479,21 +479,25 @@
...
   }
 
   &:visited {
     color: $bpk-link-visited-color;
+    color: var(--bpk-link-visited-color, $bpk-link-visited-color);
   }
 
   &:active {
     color: $bpk-link-active-color;
+    color: var(--bpk-link-active-color, $bpk-link-active-color);
```
Comment (k0nserv) [inline packages/bpk-mixins/src/mixins/_typography.scss]: We can create a mixin for this actually.

File: packages/bpk-component-link/readme.md [hunk 2]
```diff
@@ -41,3 +41,12 @@ export default () => (
 | children | -        | true     | -             |
 | onClick  | func     | true     | -             |
 | white    | bool     | false    | false         |
+
+## Theme props
+
+For both BpkLink and BpkLinkButton the following them attributes are required.
```
Comment (matthewdavidson) [inline packages/bpk-component-link/readme.md]: *theme

File: packages/bpk-component-link/src/BpkButtonLink.js [hunk 3]
```diff
@@ -19,6 +19,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { cssModules } from 'bpk-react-utils';
+import { themeAttributes } from './BpkLink';
```
Comment (matthewdavidson) [inline packages/bpk-component-link/src/BpkButtonLink.js]: `themeAttributes` should be in separate file if it's used in both?

File: packages/bpk-component-link/src/BpkLink-test.js [hunk 4]
```diff
@@ -45,4 +45,10 @@ describe('BpkLink', () => {
     const tree = renderer.create(<BpkLink href="#" id="test-id">Link</BpkLink>).toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  describe('themeAttributes', () => {
```
Comment (matthewdavidson) [inline packages/bpk-component-link/src/BpkLink-test.js]: This test exists twice ... 🙄

Comment (k0nserv) [reply packages/bpk-component-link/src/BpkLink-test.js]: Since it exported in both components

**Pull request merged**